docs/testing.md

```diff
@@ -0,0 +1,140 @@
+## Testing
+
+`tsaga` comes with its testing utilities.
+
+The utilities are similar to `redux-saga-test-plan` and modeled after how a saga would run in your application.
+
+### Sagas
+
+The assertions use the `expect` function from the `jest` testing framework.
+
+```typescript
+import { expectSaga, select, call, run, spawn } from 'tsaga';
+
+it('should run a saga', () => {
```

Comment:
Wrap in `describe('tsaga testing')` or something? I think otherwise `jest` user's would use `test` instead of `it`.